services/codacy/codacy-coverage.service.js

```diff
@@ -39,8 +39,13 @@ module.exports = class CodacyCoverage extends LegacyService {
               return
             }
             try {
-              badgeData.text[1] = res
-              badgeData.colorscheme = coveragePercentageColor(parseInt(res))
+              const coverage = parseInt(res)
+              if (isNaN(coverage)) {
```

Comment:
This might be a case where it is better to use [Number.isNaN()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN) in preference to `isNaN()`